src/modules/SdnDiag.NetworkController/public/Get-SdnInfrastructureInfo.ps1

```diff
@@ -37,7 +39,10 @@ function Get-SdnInfrastructureInfo {
         })]
         [Uri]$NcUri,
 
-        [Parameter(Mandatory = $false)]
+        [Parameter(Mandatory = $true, ParameterSetName = 'RestCertificate')]
```

Comment:
Review `Mandatory` attribute

Reply:
This is correct when using ParameterSetName and helps define what is required. It enables unique properties to be required depending on the ParameterSetName that is used.